internal/builders/execaccessbuilder/create_access_resources_test.go

```diff
@@ -134,28 +134,6 @@ var _ = Describe("RequestReconciler", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It(
```

Comment:
Why are we removing this test?

Reply:
We didn't change this behavior did we?

Reply:
This is due to the fact that we changed the `GetPodName` to `GetPod` to return an actual pod and previously returned the pod name of the .Status.PodName was set.

Reply:
We need to return back a Pod object instead we could access the Metadata further down when setting the AccessCommand 
https://github.com/diranged/oz/pull/121/files#diff-f484865b0c9ca796e879e0ce1f078021b976bf56861785f2946a6942ba8bb2d0R64

Reply:
@diranged I've added back the test and how we reconcile the podnames by the caller of the function. 

https://github.com/diranged/oz/pull/121/commits/91a491fd47b67a7aaa4b7c3150fe48e01a27fc88